scripts/stage

```diff
@@ -48,6 +48,9 @@ usage() {
 stage_s3() {
 	tarball="$(mktemp)"
 	tarball_md5="$(mktemp)"
+	tarball_manifest="$(mktemp)"
+
```

Comment:
As @jhaynes mentioned in an earlier review, this function should also have a `trap` added to remove the created temporary files.